virtwho/virt/esx/esx.py

```diff
@@ -339,7 +339,11 @@ def login(self):
         try:
             # Don't log message containing password
             logging.getLogger('suds.client').setLevel(logging.CRITICAL)
-            self.client.service.Login(_this=self.sc.sessionManager, userName=self.username, password=self.password)
+            self.client.service.Login(
+                _this=self.sc.sessionManager,
+                userName=self.username,
```

Comment:
Do we need/want to do the same to both username and password here?

Reply:
Good point 👍 . It is IMHO not allowed to use username including any non-ascii character by ESX and virt-who should log some warning, when wrong username is used, but there was bug in checking of configuration file.